frontend/packages/shared/src/types/ComponentSpecificConfig.ts

```diff
@@ -139,6 +141,8 @@ export type Summary2OverrideConfig = {
   forceShow?: boolean;
   emptyFieldText?: string;
   hideEmptyFields?: boolean;
+  isCompact?: boolean;
+  displayType?: SummaryCustomTargetType | string;
```

Comment:
I believe this can be simplified to this as `string` is already part of the type. 

```suggestion
  displayType?: SummaryCustomTargetType;
```

Reply:
Fixed